Sources/Polymorphic.swift

```diff
@@ -19,16 +20,11 @@ public protocol Polymorphic {
 
 extension Polymorphic {
     public var float: Float? {
-        guard let double = double else {
-            return nil
-        }
-
-        return Float(double)
+        return double.flatMap { Float($0) }
```

Comment:
Maybe I'm a noob but I was pretty confused by it. Depends who we think will be reading this code. Up to you.